db/migrate/20160907095103_add_availability_to_listing_shape.rb

```diff
@@ -0,0 +1,5 @@
+class AddAvailabilityToListingShape < ActiveRecord::Migration
+  def change
+    add_column :listing_shapes, :availability, :string, limit: 32, default: "none", after: :shipping_enabled
```

Comment:
You mean [ActiveRecord Enum](http://edgeapi.rubyonrails.org/classes/ActiveRecord/Enum.html)? Hmm, interesting. I have never used those. I'll check it out!